openwillis/measures/text/speech_attribute.py

```diff
@@ -4,154 +4,404 @@
+            measures["pos"],
+            measures["neg"],
+            measures["neu"],
+            measures["compound"],
+            measures["speech_mattr"],
+            measures["interrupt_flag"],
+        ]
+    )
+
+    summ_df = pd.DataFrame(
+        columns=[
+            measures["speech_minutes"],
+            measures["speech_words"],
+            measures["word_rate"],
+            measures["syllable_rate"],
+            measures["pause_rate"],
+            measures["word_pause_mean"],
+            measures["word_pause_var"],
+            measures["phrase_pause_mean"],
+            measures["phrase_pause_var"],
+            measures["speech_percentage"],
+            measures["speech_noun"],
+            measures["speech_verb"],
+            measures["speech_adj"],
+            measures["speech_pronoun"],
+            measures["pos"],
+            measures["neg"],
+            measures["neu"],
+            measures["compound"],
+            measures["speech_mattr"],
+            measures["num_turns"],
+            measures["turn_minutes_mean"],
+            measures["turn_words_mean"],
+            measures["turn_pause_mean"],
+            measures["num_one_word_turns"],
+            measures["num_interrupts"],
+        ]
+    )
+
+    return word_df, phrase_df, turn_df, summ_df
 
-def create_empty_dataframes(measures):
+
+def is_amazon_transcribe(json_conf):
     """
     ------------------------------------------------------------------------------------------------------
 
-    Creating an empty measure dataframe
+    This function checks if the json response object is from Amazon Transcribe.
 
     Parameters:
     ...........
-    measures: dict
-        config file object
+    json_conf: dict
+        JSON response object.
 
     Returns:
     ...........
-    tag_df: pandas dataframe
-        an empty dataframe for the tags
-    summ_df: pandas dataframe
-        an empty dataframe for the summary
+    bool: True if the json response object
+     is from Amazon Transcribe, False otherwise.
 
     ------------------------------------------------------------------------------------------------------
     """
-    summ_df = pd.DataFrame(columns=[measures['tot_words'], measures['speech_verb'], measures['speech_adj'],
-                                    measures['speech_pronoun'], measures['speech_noun'], measures['neg'], measures['neu'],
-                                    measures['pos'], measures['compound'], measures['speech_mattr'], measures['rate_of_speech'],
-                                    measures['pause_rate'], measures['pause_meandur'], measures['silence_ratio']])
+    return "jobName" in json_conf and "results" in json_conf
 
-    # Create an empty tag dataframe
-    tag_df = pd.DataFrame(columns=[measures['word'], measures['tag']])
-    return tag_df, summ_df
 
-def is_amazon_transcribe(json_conf):
+def filter_speaker_label(item_data, speaker_label, turns_idxs, turns, phrases_idxs, phrases):
```

Comment:
Modularization: will suggest to split this function into multiple small functions for clean code structure

Reply:
[move filter_speaker to cutil and split into smaller functions](https://github.com/bklynhlth/openwillis/pull/22/commits/b0307bad85d4e86a1a7ee1064268e2d07fd88408)